tests/common.go

```diff
@@ -6624,3 +6625,123 @@ func AddMetadataDisk(n node.Node) error {
 	return nil
 
 }
+
+// generateRandomString Creates a random string of length n and returns it
+func generateRandomString(n int) string {
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	randomString := make([]rune, n)
+	for i := range randomString {
+		randomString[i] = letters[rand.Intn(len(letters))]
+	}
+	return string(randomString)
+}
+
+// createNamespaces Create N number of namespaces and return namespace list
+func createNamespaces(numberOfNamespaces int) ([]string, error) {
```

Comment:
Is it possible to parallelize this task?

Reply:
I had a look at the `TaskHandler()` function in your PR and had a look at go routine as well. Based on this what I felt we should ideally use in the step. Say lets say you want to create 100 configmaps You can run a go rountine or the TaskHnadler which calls these functions from within the step and creates 10 configmaps at a time. This will help make sure that the k8s api isn't overloaded and also reduce time. In the core function I feel it's better to have a dumb logic.

Reply:
@kshithijiyer-px We can go ahead with the current implementation without `TaskHandler()`.
Once it is merged into master we can optimize it for faster parallel execution.

Reply:
ACK